xml/System.Runtime.CompilerServices/RuntimeHelpers.xml

```diff
@@ -767,9 +767,9 @@ This method is intended for compiler use rather than use directly in code. `T` m
       <Parameters />
       <Docs>
         <typeparam name="T">The type.</typeparam>
-        <summary>Returns a value that indicates whether the specified type is a reference type or a value type that contains references.</summary>
+        <summary>Returns a value that indicates whether the specified type is a reference type or a value type that contains references or by-refs.</summary>
```

Comment:
Does this somehow need to be qualified by tfm?

Reply:
@gewarren Do you have an opinion and potentially an example I could follow here? Would a `remarks` section be sufficient in this case?